app/nasa_collaboration/page.js

```diff
@@ -1,11 +1,12 @@
 "use client";
 
-import React, { useState, useEffect } from 'react';
-import styles from './page.module.css';
+import React, { useState, useEffect, useRef } from "react";
+import styles from "./page.module.css";
+import RoverPhoto from "./components/RoverPhoto";
 
 // Read "/app/nasa_collaboration/README.md" for more info about the API_KEY
 // You need a proper API_KEY for the requests to work
-const API_KEY = 'API_KEY';
+const API_KEY = "UL0yii13kt5FWc5tj3LlaVQnUH87BBq8t6048D6o";
```

Comment:
best practice on this case is move this part to the .env file and read it from there